packages/sdk/src/modules/base/index.ts

```diff
@@ -14,10 +14,20 @@ import { AnyTuple } from '@polkadot/types-codec/types';
 import { truncatedString } from '../../utils';
 
 type TErrorData = {
-  Module?: {
-    index?: string;
-    error?: string;
-  };
+  dispatchError: {
```

Comment:
Needed to have 'dispatchError' in the object in order to successfully read the error. 'dispatchInfo' contains more information, but not relevant to be logged to the user.